.github/workflows/code-quality.yml

```diff
@@ -0,0 +1,34 @@
+name: Assess code quality
```

Comment:
Perhaps we could change the script and the name in the future. I don't have a suggestion right now. 😉